src/ai/llm/services/anthropic.ts

```diff
@@ -18,6 +21,8 @@ export class AnthropicService implements ILLMService {
     private clientManager: ClientManager;
     private messageManager: MessageManager;
     private eventEmitter: EventEmitter;
+    private systemPromptBuilder?: SystemPromptBuilder;
+    private promptContext?: PromptContext;
```

Comment:
now both the llmservice and the messagemanager have systemPromptBuilder and promptContext - maybe there is a better way of doing this? would like to know thoughts